pkg/util/contextUtils_test.go

```diff
@@ -32,7 +32,7 @@ func TestRetryUntilSuccessOrCancellation(t *testing.T) {
 	})
 
 	t.Run("ok after retry", func(t *testing.T) {
-		testCtx, cancel := context.WithTimeout(context.Background(), 1*time.Second)
+		testCtx, cancel := context.WithTimeout(context.Background(), 2*time.Second)
```

Comment:
what is up with this change?

Reply:
Due to high CPU usage the test was so slow that one second was not enough and the test failed. Nevertheless, The timeout should be as low as possible as this is the time the test needs at maximum if a bug occurred in the code.

Reply:
Hmmm I feel like high CPI usage was not necessarily the problem. When the context timeout 1 second is and the retry interval is also 1 second than we have something like a race conditions, don't we? 

Can we just make the retry-Interval very small? that would minimize the runtime of the test as well. Like 1 milisecond instead of 1 second?

Reply:
Ah, you are correct! I thought it would do a retry immediately. Now i am wondering why this test worked for nine months without any problems. I fixed it in the latest commit.